frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js

```diff
@@ -4,7 +4,7 @@ import Typography from "@mui/material/Typography";
 export default function CircularProgressWithLabel(props) {
   return (
     <Box sx={{ position: "relative", display: "inline-flex" }}>
-      <CircularProgress variant="determinate" {...props} style={{ color: "var(--teal-green)" }} />
+      <CircularProgress variant="determinate" value={props.value} style={{ color: "var(--teal-green)" }} />
```

Comment:
why was this necessary?

Reply:
This removed an annoying error message from the console. It is not present in production but fires off every page load when testing.

Reply:
You can also set default props and do `CircularProgressWithLabel({ value, colorOfTextLabel = "black" })` so remove the extra code and conditional